tests/app/questionnaire/test_path_finder.py

```diff
@@ -366,21 +366,6 @@ def test_next_location_empty_routing_rules(self):
 
         self.assertEqual(next_location, expected_next_location)
 
-    def test_interstitial_post_blocks(self):
```

Comment:
This test fails after adding default routes to star_wars

I can't see any reason that the summary shouldn't be on the path. It seems this has only been passing because of the missing default route. There are also no interstitials in star wars.